docs/mobile-apps/automated-testing/ipa-files.md

```diff
@@ -88,3 +89,26 @@ This will also overwrite the **Build Settings** at the Project level to that iOS
 4. In the **Product** menu, select **Build**.
 5. Navigate to your Xcode project's **Products** directory and find the generated **.app** file.
 6. Compress the **.app** file into an archive (**.zip** file). The **.zip** file is now ready to upload to Sauce Labs.
+
+## Sauce Resigning Enablements
+
+Sauce Labs applies its own resigning to apps that are installed on our public iOS devices. Our resigner includes the following `keychain-access-groups` entitlements:
+
+|Key|Value|
+|------|--------------|
+|`application-identifier` | `<string>XXXXXXXXXX.*</string>` |
+|	`keychain-access-groups` | `<array>`<br/>&nbsp;&nbsp;`<string>XXXXXXXXXX.*</string>`<br/>&nbsp;&nbsp;`  <string>com.apple.token</string>`<br/>`</array>` |
+|`get-task-allow` | `<true/>` |
+|`com.apple.developer.team-identifier` | `<string>XXXXXXXXXX</string>` |
+|`com.apple.developer.ubiquity-kvstore-identifier` | `<string>XXXXXXXXXX.*</string>` |
+|`com.apple.developer.ubiquity-container-identifiers` | `<array>`<br/>&nbsp;&nbsp;`<string>XXXXXXXXXX.*</string>`<br/>`</array>` |
+|`inter-app-audio` | `<true/>` |
+|`com.apple.developer.networking.networkextension` | `<array>`<br/> &nbsp;&nbsp;`<string>app-proxy-provider</string>`<br/>&nbsp;&nbsp;`<string>content-filter-provider</string>`<br/> &nbsp;&nbsp;`<string>packet-tunnel-provider</string>`<br/>&nbsp;&nbsp;`<string>dns-proxy</string>`<br/> &nbsp;&nbsp;`<string>dns-settings</string>`<br/>`</array>` |
+|`com.apple.developer.siri` | `<true/>` |
+|`com.apple.developer.pass-type-identifiers` | `<array>`<br/>&nbsp;&nbsp;   `<string>XXXXXXXXXX.*</string>`<br/>`</array>` |
+
+On Sauce Labs private devices, you may choose to resign using your own provisioning profile, in which case, we support all entitlements in:
```

Comment:
> ...you may choose to resign using your own provisioning profile,.....

This sentence is not really clear, when you disable the feature we DON'T resign, people just use their own signing. can we make this sentence a bit more clear, and also tell / add a link what people need to do when they disable resigning?

Reply:
ah yeah that is unclear. We also want to advertise that Private Devices are an option if you need specific entitlements

Reply:
Ok. Let me see if I can clarify those points. Thanks, guys.